Common/Utilities.cs

```diff
@@ -557,2637 +559,2638 @@ public static void Print(IServiceBusNamespace serviceBusNamespace)
-            }
-
-
-            var networkProfile = new StringBuilder().Append("\n\tNetworkProfile: ");
-            foreach (var networkInterfaceId in virtualMachine.NetworkInterfaceIds)
-            {
-                networkProfile.Append("\n\t\tId:").Append(networkInterfaceId);
-            }
-
-            var msi = new StringBuilder().Append("\n\tMSI: ");
-            msi.Append("\n\t\tMSI enabled:").Append(virtualMachine.IsManagedServiceIdentityEnabled);
-            msi.Append("\n\t\tMSI Active Directory Service Principal Id:").Append(virtualMachine.SystemAssignedManagedServiceIdentityPrincipalId);
-            msi.Append("\n\t\tMSI Active Directory Tenant Id:").Append(virtualMachine.SystemAssignedManagedServiceIdentityTenantId);
-
-            Utilities.Log(new StringBuilder().Append("Virtual Machine: ").Append(virtualMachine.Id)
-                    .Append("Name: ").Append(virtualMachine.Name)
-                    .Append("\n\tResource group: ").Append(virtualMachine.ResourceGroupName)
-                    .Append("\n\tRegion: ").Append(virtualMachine.Region)
-                    .Append("\n\tTags: ").Append(FormatDictionary(virtualMachine.Tags))
-                    .Append("\n\tHardwareProfile: ")
-                    .Append("\n\t\tSize: ").Append(virtualMachine.Size)
-                    .Append(storageProfile)
-                    .Append(osProfile)
-                    .Append(networkProfile)
-                    .Append(msi)
-                    .ToString());
-        }
-
-        public static void PrintStorageAccountKeys(IReadOnlyList<StorageAccountKey> storageAccountKeys)
-        {
-            foreach (var storageAccountKey in storageAccountKeys)
-            {
-                Utilities.Log($"Key {storageAccountKey.KeyName} = {storageAccountKey.Value}");
-            }
-        }
-
-        public static void PrintStorageAccount(IStorageAccount storageAccount)
-        {
-            Utilities.Log($"{storageAccount.Name} created @ {storageAccount.CreationTime}");
-        }
-
-        public static string CreateRandomName(string namePrefix)
-        {
-            return SdkContext.RandomResourceName(namePrefix, 30);
-        }
-
-        public static void PrintAvailabilitySet(IAvailabilitySet resource)
-        {
-            Utilities.Log(new StringBuilder().Append("Availability Set: ").Append(resource.Id)
-                .Append("Name: ").Append(resource.Name)
-                .Append("\n\tResource group: ").Append(resource.ResourceGroupName)
-                .Append("\n\tRegion: ").Append(resource.Region)
-                .Append("\n\tTags: ").Append(FormatDictionary(resource.Tags))
-                .Append("\n\tFault domain count: ").Append(resource.FaultDomainCount)
-                .Append("\n\tUpdate domain count: ").Append(resource.UpdateDomainCount)
-                .ToString());
-        }
+        //public static void PrintVirtualMachineCustomImage(IVirtualMachineCustomImage image)

```

Comment:
I commented out some useless code of this project now, and wondering if it is ok to **delete** them? 
This Utilties.cs file exist in many samples  (For example, all Service Bus sample project has this file). But like this project, not of all code needed for each project.  Some project just need two or three methods in Utilties.cs file.